src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,37 @@
+# Download and execute Chocolatey install script
+Set-ExecutionPolicy Bypass -Scope Process -Force
+
+# Enable TLS 1.2 support for .NET framework using the security protocal type 3072
```

Comment:
Fix typo:
```suggestion
# Enable TLS 1.2 support for .NET framework using the security protocol type 3072
```

Reply:
Also, can you please add a MS (or other authoritative) documentation URL that shows that 3072 means TLS 1.2?

Reply:
@dav3r resolved with [10afc9f](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/10afc9f3a3ea050ee3fffd388f4da37b66359fd3)